test/helpers.test.js

```diff
@@ -149,16 +149,18 @@ describe('Helpers', () => {
     });
```

Comment:
not your work but indentation seems a bit weird for example on line 128

Reply:
fixed :)